index.html

```diff
@@ -41,8 +41,167 @@ <h1 class="primary">
                </ul>
             </div>
             
-            <img src="assets/bottom arrow.svg" alt="bottom-arrow">
         </section>
+        
+        <div class="bottom-arrow">
+            <img src="assets/bottom arrow.svg" alt="bottom-arrow" >
+        </div>
+        
+        <section class="heading">
```

Comment:
Great work on implementing this part, Kindly I suggest you stick to the design as much as possible to make your project look more professional.

According to the Figma design, Kindly adjust the padding and margin to make your design as it is in Figma.

> ## The Figma Design
>
><img width="384" alt="Screenshot 2023-06-03 at 12 24 25 PM" src="https://github.com/BHObserver/portfolio/assets/91237525/d3bf096c-0cfc-484e-ae4f-8689951178fb">
>
> ## Your Desiign
>
><img width="372" alt="Screenshot 2023-06-03 at 12 25 05 PM" src="https://github.com/BHObserver/portfolio/assets/91237525/578f5d69-f8da-4f8c-b63d-51ec8f882aef">